src/client/client-load-module.ts

```diff
@@ -29,7 +38,7 @@ export const loadModule = (
     /* webpackInclude: /\.entry\.js$/ */
     /* webpackExclude: /\.system\.entry\.js$/ */
     /* webpackMode: "lazy" */
-    `./${bundleId}.entry.js${BUILD.hotModuleReplacement && hmrVersionId ? '?s-hmr=' + hmrVersionId : ''}`
+    `${MODULE_IMPORT_PREFIX}${bundleId}.entry.js${BUILD.hotModuleReplacement && hmrVersionId ? '?s-hmr=' + hmrVersionId : ''}`
```

Comment:
This is a weird hack I posted in Slack about - my understanding is that when Esbuild sees a dynamic import it will attempt to figure out if it can resolve the file that the `import` corresponds to and, if it decides it is an 'analyzable import', it will bundle the file if so. This behavior is documented here: https://esbuild.github.io/api/#non-analyzable-imports

One of the consequences of this was that since our template literal here began with `./`, Esbuild decided this was an 'analyzable import' and tried to bundle in the file it referred to, but because the template is a runtime construct it wasn't resolving anything. This led to an error where the `loadModule` function here wasn't working correctly in the lazy build.

Pulling out the `"./"` string as a separate 'prefix' constant addressed the issue by making the `import()` look 'non-analyzable' to Esbuild, so it just leaves it alone and we retain the runtime behavior that we want.